core/commands/stat.go

```diff
@@ -0,0 +1,172 @@
+package commands
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"time"
+
+	humanize "github.com/jbenet/go-ipfs/Godeps/_workspace/src/github.com/dustin/go-humanize"
+
+	cmds "github.com/jbenet/go-ipfs/commands"
+	metrics "github.com/jbenet/go-ipfs/metrics"
+	peer "github.com/jbenet/go-ipfs/p2p/peer"
+	protocol "github.com/jbenet/go-ipfs/p2p/protocol"
+	u "github.com/jbenet/go-ipfs/util"
+)
+
+var StatsCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline:          "Query IPFS daemon statistics",
+		ShortDescription: ``,
+	},
+
+	Subcommands: map[string]*cmds.Command{
+		"bw":      statBwCmd,
+		"bw-poll": statBwPollCmd,
```

Comment:
the structure of the command changes a *lot* to do the polling. I could do it as just one command and a flag, but it wouldnt be as pretty